onnxruntime/core/framework/tunable.h

```diff
@@ -256,7 +259,13 @@ class TunableOp {
       }
 
       WarmUp(candidate, params);
-      auto time = Profile(candidate, params);
+
+      auto approx_duration = Profile(candidate, params, approx_num_iter);
```

Comment:
I think use a hardcoded, like `2`, probe iters instead of `approx_num_iter` heuristics will work just fine.